core/src/main/scala/kafka/message/Message.scala

```diff
@@ -206,6 +293,33 @@ class Message(val buffer: ByteBuffer) {
    * The attributes stored with this message
    */
   def attributes: Byte = buffer.get(AttributesOffset)
+
+  /**
+   * The timestamp of the message, only available when the "magic" value is greater than 0
+   * When magic > 0, The timestamp of a message is determined in the following way:
+   * 1. TimestampType = None and wrapperMessageTimestamp is None - Uncompressed message, timestamp and timestamp type are in the message.
```

Comment:
Sorry for the confusion, "TimestampType" in the comments should actually be "wrapperMessageTimestampType".